src/main/java/fibonacciSequence/FibonacciSequenceDefinition.java

```diff
@@ -0,0 +1,29 @@
+package fibonacciSequence;
+
+import java.util.*;
+
+public class FibonacciSequenceDefinition {
+    private int number = 0;
+    private List<Integer> fibonacciSequence = new ArrayList<>();
+
+    public List<Integer> defineFibonacciSequence(int number) {
+        this.number = number;
+        int sum;
+
+        if (number == 1) {
+            fibonacciSequence.add(1);
+            return fibonacciSequence;
+        } else if (number >= 2) {
+            fibonacciSequence.add(1);
+            fibonacciSequence.add(1);
+
+            for (int i = 2; i < number; i++) {
+                sum = fibonacciSequence.get(i - 1) + fibonacciSequence.get(i - 2);
+                fibonacciSequence.add(sum);
```

Comment:
could be simplified

Reply:
Optimized in b7f7b7eb3c7316a5b476c68f07f63ae64ca984d3. 
Changed to Stream.iterate()...